### PR TITLE
Feature/#94 mltc benchmark

### PR DESCRIPTION
**Pull request by duke-b**
# Description

### Issue

This PR addresses issue #[issue-number].

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [ ] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/DevoteamNL/opendora/blob/main/CONTRIBUTING.md).**
- [x] **I have ran the linting and formatting scripts to make sure the code is consistently styled.**
- [x] **I have built/compiled and ran the tests to make sure the code works correctly.**
- [x] **I have added or changed tests to cover new or different functionality, where applicable.**
- [ ] **Create an issue and link to the pull request.**

Note: Keeping the PR small and focused helps make it easier to review and merge. If you have multiple changes you want to make, please consider submitting them as separate pull requests.

### CODE REVIEW

File: devlake-go/api/main.go [hunk 1]
```diff
@@ -55,9 +55,10 @@ func metricHandler(client sql_client.ClientInterface) func(w http.ResponseWriter
...
 func benchmarkHandler(client sql_client.ClientInterface) func(w http.ResponseWriter, r *http.Request) {
-	dfService := service.BenchmarkDfService{Client: client}
+	benchmarkService := service.BenchmarkService{Client: client}
 	serviceMap := map[string]service.Service[models.BenchmarkResponse]{
-		"df": dfService,
+		"df":   benchmarkService,
+		"mltc": benchmarkService,
 	}
 
 	return handler(validation.ValidBenchmarkServiceParameters, serviceMap)
```
Comment (kylejwatson) [inline devlake-go/api/main.go:64]: I think you'll need to change the `ValidBenchmarkServiceParameters` function to also accept 'mltc' as a type.

File: devlake-go/api/sql_client/sql_queries/benchmark_mltc.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+_median_change_lead_time as(
+-- use median PR cycle time as the median change lead time
+    SELECT max(pr_cycle_time) as median_change_lead_time
+    FROM _median_change_lead_time_ranks
+    WHERE ranks <= 0.5
+)
+
+SELECT
+  CASE
+    WHEN median_change_lead_time < 60 then "Less than one hour"
```
Comment (kylejwatson) [inline devlake-go/api/sql_client/sql_queries/benchmark_mltc.sql:36]: Just realised, shouldn't this be just a key that we then translate on the frontend? Similar to df with 'on-demand', 'week-month',  'month-6month' and 'lt-6month'

Comment (kylejwatson) [reply devlake-go/api/sql_client/sql_queries/benchmark_mltc.sql:36]: And for the else we could just use 'none' or something

Comment (duke-b) [reply devlake-go/api/sql_client/sql_queries/benchmark_mltc.sql:36]: Very good catch

**Pull request merged**